tests/test_birdseye.py

```diff
@@ -275,6 +275,26 @@ def f():
                       ['3', ["'Hello World!H...d!Hello World!'",
                              t['str'], 'len() = 600']]]]])
 
+    def test_decorate_class(self):
+        def fooz(self):
+            return 'method outside class'
+
+        @eye
+        class Testclass(object):
+
+            call_meth = fooz
+            def barz(self):
+                return 'class decorator test'
+
+        def get_stuff(method):
+            return get_call_stuff(get_call_ids(method)[0]).call
+
+        x = Testclass()
+        call = get_stuff(x.barz)
+        self.assertEqual("'class decorator test'", call.return_value)
```

Comment:
this is still duplicated, put the assert inside the local function and rename the function.

Reply:
@alexmojaki  I made the updates.